src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -2164,6 +2164,121 @@ class C {
                 SymbolDisplayPartKind.Keyword);
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/77219")]
+        public void TestPropertyBackingField_Minimal_01()
+        {
+            var text = @"
+#nullable enable
+class C {
+    string P { get; set; } }
+";
+
+            Func<NamespaceSymbol, Symbol> findSymbol = global =>
+                global.GetTypeMembers("C", 0).Single().
+                GetMembers("<P>k__BackingField").Single();
```

Comment:
There's no better way to do this?

Reply:
There isn't an API on the IPropertySymbol that gives us the backing field, for example, so I don't think there are particularly better ways to do this in general.

Reply:
I believe there's an issue asking for this. The ide has wanted such an associated symbol API since forever :-)

Note, what you can do instead is get all the fields and they get the single one associated with the property of interest.